### PR TITLE
4.6.x imos gitaction

### PR DESCRIPTION
**Pull request by utas-raymondng**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    open-pull-requests-limit: 0
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (digorgonzola) [inline .github/dependabot.yml]: new line required

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            git clean --force -xd
+            ./gradlew clean assemble
+          '
+
+      - name: Archive Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: war-file
+          path: 'tds/build/libs/*.war'
+          if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn`
```
Comment (digorgonzola) [inline .github/workflows/build.yml]: new line

File: Dockerfile.build [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    ca-certificates \
+    git \
+    openjdk-8-jdk \
+    python3-dev \
+    python3-pip \
+    maven \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```
Comment (digorgonzola) [inline Dockerfile.build]: new line here

**Review by digorgonzola — CHANGES_REQUESTED** [review]

File: Dockerfile.build (hunk 3 above)
Comment (utas-raymondng) [reply Dockerfile.build]: added

File: .github/workflows/build.yml (hunk 2 above)
Comment (utas-raymondng) [reply .github/workflows/build.yml]: added

File: .github/dependabot.yml (hunk 1 above)
Comment (utas-raymondng) [reply .github/dependabot.yml]: added

**Review by digorgonzola — APPROVED** [review]

**Pull request merged**